Index: privoxy-3.0.28-stable/cgi.c
===================================================================
--- privoxy-3.0.28-stable/cgi.c
+++ privoxy-3.0.28-stable/cgi.c
@@ -1089,19 +1089,17 @@
 {
    memset(cgi_error_memory_response, '\0', sizeof(*cgi_error_memory_response));
    cgi_error_memory_response->head =
-      "HTTP/1.0 500 Internal Privoxy Error\r\n"
+      "HTTP/1.0 500 Internal Error\r\n"
       "Content-Type: text/html\r\n"
       "\r\n";
    cgi_error_memory_response->body =
       "<html>\n"
       "<head>\n"
-      " <title>500 Internal Privoxy Error</title>\n"
-      " <link rel=\"shortcut icon\" href=\"" CGI_PREFIX "error-favicon.ico\" type=\"image/x-icon\">"
+      " <title>500 Internal Error</title>\n"
       "</head>\n"
       "<body>\n"
-      "<h1>500 Internal Privoxy Error</h1>\n"
-      "<p>Privoxy <b>ran out of memory</b> while processing your request.</p>\n"
-      "<p>Please contact your proxy administrator, or try again later</p>\n"
+      "<h1>500 Internal Error</h1>\n"
+      "<p><b>Ran out of memory</b></p>\n"
       "</body>\n"
       "</html>\n";
 
@@ -1159,29 +1157,17 @@
                              const char *template_name)
 {
    static const char status[] =
-      "500 Internal Privoxy Error";
+      "500 Internal Error";
    static const char body_prefix[] =
       "<html>\n"
       "<head>\n"
-      " <title>500 Internal Privoxy Error</title>\n"
-      " <link rel=\"shortcut icon\" href=\"" CGI_PREFIX "error-favicon.ico\" type=\"image/x-icon\">"
+      " <title>500 Internal Error</title>\n"
       "</head>\n"
       "<body>\n"
-      "<h1>500 Internal Privoxy Error</h1>\n"
-      "<p>Privoxy encountered an error while processing your request:</p>\n"
-      "<p><b>Could not load template file <code>";
+      "<h1>500 Internal Error</h1>\n"
+      "<p><b>Could not load <code>";
    static const char body_suffix[] =
-      "</code> or one of its included components.</b></p>\n"
-      "<p>Please contact your proxy administrator.</p>\n"
-      "<p>If you are the proxy administrator, please put the required file(s)"
-      "in the <code><i>(confdir)</i>/templates</code> directory.  The "
-      "location of the <code><i>(confdir)</i></code> directory "
-      "is specified in the main Privoxy <code>config</code> "
-      "file.  (It's typically the Privoxy install directory"
-#ifndef _WIN32
-      ", or <code>/etc/privoxy/</code>"
-#endif /* ndef _WIN32 */
-      ").</p>\n"
+      "</code></b></p>\n"
       "</body>\n"
       "</html>\n";
    const size_t body_size = strlen(body_prefix) + strlen(template_name) + strlen(body_suffix) + 1;
@@ -1242,22 +1228,17 @@
                          jb_err error_to_report)
 {
    static const char status[] =
-      "500 Internal Privoxy Error";
+      "500 Internal Error";
    static const char body_prefix[] =
       "<html>\n"
       "<head>\n"
-      " <title>500 Internal Privoxy Error</title>\n"
-      " <link rel=\"shortcut icon\" href=\"" CGI_PREFIX "error-favicon.ico\" type=\"image/x-icon\">"
+      " <title>500 Internal Error</title>\n"
       "</head>\n"
       "<body>\n"
-      "<h1>500 Internal Privoxy Error</h1>\n"
-      "<p>Privoxy encountered an error while processing your request:</p>\n"
+      "<h1>500 Internal Error</h1>\n"
       "<p><b>Unexpected internal error: ";
    static const char body_suffix[] =
       "</b></p>\n"
-      "<p>Please "
-      "<a href=\"http://sourceforge.net/tracker/?group_id=11118&amp;atid=111118\">"
-      "file a bug report</a>.</p>\n"
       "</body>\n"
       "</html>\n";
    /* Includes room for larger error numbers in the future. */
