Index: privoxy-3.0.28-stable/doc/source/p-config.sgml
===================================================================
--- privoxy-3.0.28-stable/doc/source/p-config.sgml
+++ privoxy-3.0.28-stable/doc/source/p-config.sgml	2018-12-31 10:47:45.797289992 +0100
@@ -295,7 +295,7 @@
  </varlistentry>
 </variablelist>
 
-<![%config-file;[<literallayout>@@#user-manual https://www.privoxy.org/user-manual/</literallayout>]]>
+<![%config-file;[<literallayout>@@#user-manual /usr/share/doc/privoxy/user-manual/</literallayout>]]>
 </sect3>
 
 
@@ -517,7 +517,7 @@
  </varlistentry>
 </variablelist>
 
-<![%config-file;[<literallayout>@@confdir .</literallayout>]]>
+<![%config-file;[<literallayout>@@confdir /etc/privoxy</literallayout>]]>
 </sect3>
 
 <!--   ~~~~~       New section      ~~~~~     -->
@@ -657,7 +657,7 @@
  </varlistentry>
 </variablelist>
 
-<![%config-file;[<literallayout>@@logdir .</literallayout>]]>
+<![%config-file;[<literallayout>@@logdir /var/log/privoxy</literallayout>]]>
 </sect3>
 
 
@@ -729,7 +729,9 @@
 
 <!-- NOTE: alternate markup to make a simpler list doesn't work due to -->
 <!-- html -> text conversion, blah -->
-<![%config-file;[<literallayout>@@actionsfile match-all.action # Actions that are applied to all sites and maybe overruled later on.</literallayout>]]>
+<![%config-file;[<literallayout>@@actionsfile match-all.action # Actions that are applied to all </literallayout>]]>
+<![%config-file;[<literallayout>@@                             # sites and maybe overruled later </literallayout>]]>
+<![%config-file;[<literallayout>@@                             # on.</literallayout>]]>
 <![%config-file;[<literallayout>@@actionsfile default.action   # Main actions file</literallayout>]]>
 <!--
  XXX: Like user.filter, user.action should probably be commented out
@@ -1065,8 +1067,10 @@
  </varlistentry>
 </variablelist>
 
-<![%config-file;[<literallayout>@@#debug     1 # Log the destination for each request &my-app; let through.</literallayout>]]>
-<![%config-file;[<literallayout>@@#debug  1024 # Log the destination for requests &my-app; didn't let through, and the reason why.</literallayout>]]>
+<![%config-file;[<literallayout>@@#debug     1 # Log the destination for each request &my-app; </literallayout>]]>
+<![%config-file;[<literallayout>@@#            # lets through.</literallayout>]]>
+<![%config-file;[<literallayout>@@#debug  1024 # Log the destination for requests &my-app; didn't </literallayout>]]>
+<![%config-file;[<literallayout>@@#            # let through, and the reason why.</literallayout>]]>
 <![%config-file;[<literallayout>@@#debug  4096 # Startup banner and warnings</literallayout>]]>
 <![%config-file;[<literallayout>@@#debug  8192 # Non-fatal errors</literallayout>]]>
 </sect3>
